src/StatusQ/Core/Utils/Utils.qml

```diff
@@ -151,6 +153,71 @@ QtObject {
         }
     }
 
+    function linkifyAndXSS(inputText) {
+        //URLs starting with http://, https://, or ftp://
+        var replacePattern1 = /(\b(https?|ftp|statusim):\/\/[-A-Z0-9+&@#\/%?=~_|!:,.;]*[-A-Z0-9+&@#\/%=~_|])/gim;
+        var replacedText = inputText.replace(replacePattern1, "<a href='$1'>$1</a>");
+
+        //URLs starting with "www." (without // before it, or it'd re-link the ones done above).
+        var replacePattern2 = /(^|[^\/])(www\.[\S]+(\b|$))/gim;
+        replacedText = replacedText.replace(replacePattern2, "$1<a href='http://$2'>$2</a>");
+
+        return XSS.filterXSS(replacedText)
+    }
+
+    function filterXSS(inputText) {
+        return XSS.filterXSS(inputText)
+    }
+
+    function getMessageWithStyle(msg, hoveredLink = "") {
+        return `<style type="text/css">` +
```

Comment:
I'm not 100% sure, but probably it could be done by using single pair of ``. Then all + could be removed making it cleaner.